src/receive_imf/tests.rs

```diff
@@ -3244,11 +3248,11 @@ async fn test_weird_and_duplicated_filenames() -> Result<()> {
         "a. tar.tar.gz",
     ] {
         let attachment = alice.blobdir.join(filename_sent);
-        let content = format!("File content of {filename_sent}");
+        let content = "File content of tar.gz archive".to_string();
```

Comment:
```suggestion
        let content = "Bli Bla Blup".to_string();
```

Reply:
If I change this, I'll also have to change the hash below - what's the difference?

Reply:
We change the file name but the message remains "File content of tar.gz" Only nitpicking especially as it only is a test